examples/apps/heart_disease_eval/client/heart_gui.py

```diff
@@ -0,0 +1,493 @@
+# Copyright 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+import random
+import json
+import argparse
+import logging
+import base64
+
+import tkinter as tk
+import tkinter.messagebox as messagebox
+import tkinter.font as font
+
+from service_client.generic import GenericServiceClient
+import utility.utility as enclave_helper
```

Comment:
enclave_helper doesn't seem to a good name for utility.utility. 
Let's keep it import utility.utility as utility

Reply:
Done.